app/views/layouts/overdue.html.erb

```diff
@@ -14,6 +14,21 @@
         <i class="fas fa-filter"></i> Search filters
       </a>
     </div>
+
+    <section class="mb-4">
```

Comment:
Oh, there's no download link on desktop either right now?

Reply:
There is, it's downstairs, under the `desktop nav` section.